go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/btcsuite/btcd v0.22.1
 	github.com/containerd/cgroups v1.0.4 // indirect
 	github.com/elastic/gosigar v0.14.2 // indirect
+	github.com/envoyproxy/protoc-gen-validate v0.9.1
```

Comment:
It looks like the updated dep is this https://github.com/bufbuild/protoc-gen-validate